lib/request_handler/body_parser.rb

```diff
@@ -22,6 +23,10 @@ def run
 
     private
 
+    def jsonapi?
+      request.env['Content-Type'] == 'application/vnd.api+json'
```

Comment:
I think we need a way to override this - not sure if our clients correctly send this content-type right now

Reply:
you mean setting 'application/vnd.api+json' as a default?

Reply:
The jsonapi object also indicates plain json
http://jsonapi.org/format/#document-jsonapi-object
Maybe it's enough to rely on matching content_type OR jsonapi object being present?

Reply:
jsonapi object was my first thought, but this is not a good idea, as it's optional and not required.
we will go for another solution -> to set the type in the service's config

Reply:
What would that look like? It should be possible to have jsonapi and plain json endpoints in the same service!

Reply:
Please take a look at the new commit.
The type "jsonapi" or "json" will now be passed in the handler configuration.